modules/quickstart-uyuni/pages/install-uyuni-server.adoc

```diff
@@ -111,7 +111,6 @@ Do not share the main administration access details.
 .Procedure: Setting Up the Main Administration Account
 . In your web browser, enter the address for the {productname} {webui}.
   This address was provided after you completed setup.
-  For more information, see xref:quickstart-uyuni:setup-uyuni-server.adoc#server-setup-yast[].
 . Log in to the {webui}, navigate to the menu:Create Organization[Organization Name] field, and enter your organization name.
 . In the menu:Create Organization[Desired Login] and menu:Create Organization[Desired Password] fields, enter your username and password.
 . Fill in the Account Information fields including an email for system notifications.
```

Comment:
Wondering whether we want to keep this capitalized?  I think it is the title of the dialog.  If yes, we want adoc markup, probably [guimenu].